.github/workflows/update-core-deps.yml

```diff
@@ -34,13 +41,13 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.TS_GITHUB_BOT_AUTH }}
       - id: git-diff
-        run: git diff --quiet HEAD baselines
+        run: git diff --quiet HEAD baselines submodules
```

Comment:
There's no path `submodules`, and just comparing `baselines` is enough, nothing more is needed here.